src/inspector/models/general/appearance/appearancesettingsmodel.cpp

```diff
@@ -142,7 +148,10 @@ void AppearanceSettingsModel::loadProperties(const PropertyIdSet& propertyIdSet)
     }
 
     if (muse::contains(propertyIdSet, Pid::Z)) {
-        loadPropertyItem(m_arrangeOrder);
+        loadPropertyItem(m_arrangeOrder, m_elementsForArrangeProperty);
+        if (m_arrangeOrder) {
+            m_arrangeOrder->setIsEnabled(!m_elementsForArrangeProperty.empty());
```

Comment:
I was wondering whether this should instead take place inside the early return of `AbstractInspectorModel::loadPropertyItem` (if `elements` is empty, disable the property item). Feels like it could be a slightly risky change, though...

Reply:
I think that would make sense, and I can't imagine it will have catastrophic consequences.